pangram.js

```diff
@@ -7,14 +7,20 @@
 'use strict';
 
 var Pangram = function(candidate) {
```

Comment:
**_AMAZING! INCREDIBLE! :open_mouth:_**

But you're solution is **exactly the same** that's the one which is give on the official [exercism.io](http://exercism.io/) repository :broken_heart: ..